addons/analytic/views/analytic_distribution_model_views.xml

```diff
@@ -29,7 +29,7 @@
                                 <field name="company_id" groups="base.group_multi_company"/>
                             </group>
                         </group>
-                        <group string="Analytic distribution to apply" colspan="2">
+                        <group string="Distribution to apply" colspan="1">
                             <field name="analytic_distribution" widget="analytic_distribution"
                                    options="{'force_applicability': 'optional', 'disable_save': true}"/>
                         </group>
```

Comment:
While this does work, it also reduces the group (obviously) by half so it stops at the middle.
I think it would be better to just put limit the field width itself by 50%
```suggestion
                        <group string="Distribution to apply" colspan="1">
                            <field name="analytic_distribution" widget="analytic_distribution"
                                   options="{'force_applicability': 'optional', 'disable_save': true}"
                                   class="w-50"/>
                        </group>
```